requirements.txt

```diff
@@ -2,5 +2,5 @@ pre-commit
 filecheck
 lit
 numpy
-xdsl @ git+https://github.com/xdslproject/xdsl.git@5bd448e56719508b9f3149167430f0a8b7c66574
+xdsl @ git+https://github.com/xdslproject/xdsl.git@2d2e3e5f274210e8f19b03fc98c4033fd0bc3ee4
```

Comment:
```suggestion
xdsl @ git+https://github.com/xdslproject/xdsl.git@168758fee7fda1beec84bab81b249542d76fa188
```

Reply:
Let's use the most recent version as we just merged a nice fix for something else :)